iphone/Classes/TiUIWebView.m

```diff
@@ -671,6 +671,12 @@ -(CGFloat)contentWidthForWidth:(CGFloat)value
     return ret;
 }
 
+- (void)setKeyboardDisplayRequiresUserAction_:(id)value
+{
+    [[self proxy] replaceValue:value forKey:@"keyboardDisplayRequiresUserAction" notification:NO];
+    webview.keyboardDisplayRequiresUserAction = [TiUtils boolValue:value def:YES];
+}
```

Comment:
And please implement the getter in the proxy + add unit tests in [titanium-mobile-mocha-suite](https://github.com/appcelerator/titanium-mobile-mocha-suite)